clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

```diff
@@ -192,10 +192,17 @@ public static Errors forCode(short code) {
     }
 
     /**
-     * Return the error instance associated with this exception (or UNKNOWN if there is none)
+     * Return the error instance associated with this exception or any of its superclasses (or UNKNOWN if there is none)
```

Comment:
It may be worth explaining what happens if both a subclass and superclass have a mapping (the subclass mapping is used).